Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2452,7 +2452,7 @@ GetUserMediaRequiresFocus:
 
 GoogleAntiFlickerOptimizationQuirkEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
Again, `shipping` would lose the UI toggle we used to have, which might be useful for debugging. `stable` retains the toggle.